lib/hawkular/operations/operations_api.rb

```diff
@@ -335,7 +354,6 @@ def invoke_operation_helper(operation_payload, operation_name = nil, binary_cont
     end
 
     def check_pre_conditions(hash = {}, params = [], &callback)
-      fail 'Handshake with server has not been done.' unless @ws.open?
```

Comment:
Why is this statement removed? I'm pretty sure there's a reason but following the changes in the PR doesn't point on it very clearly to me...